addons/udes_stock/views/stock_picking.xml

```diff
@@ -127,6 +127,11 @@
                 <attribute name="invisible">1</attribute>
             </xpath>
 
+            <!--Add Set To Draft button visible when picking has been marked as to do-->
+            <xpath expr="//button[@name='action_confirm']" position="before">
+                <button name="button_draft" attrs="{'invisible': [('state', 'not in', ['waiting', 'confirmed'])]}" string="Set To Draft" type="object" class="oe_highlight" groups="base.group_user"/>
```

Comment:
Sorry last one!

Just a suggestion:

1. Remove `class="oe_highlight"` - as this isn't a happy path or natural progression for the picking (same like how unreserve button isn't highlighted)
2. Put the Set to Draft button at the end, like the unreserve button is, again because it isn't a natural progression for the picking